app/controllers/requests/received_requests_controller.rb

```diff
@@ -1,4 +1,8 @@
 class Requests::ReceivedRequestsController < ApplicationController
   def index
+    receivers = Receiver.where(user: current_user)
+    @requests = []
+    receivers.each { |receiver| @requests << receiver.request }
+    @requests
```

Comment:
but then, I guess you just need the requests. so we don't need the receivers.

so I would erase all that, and just keep:

```
@requests = current_user.requests
```

But then you need to define a scope, to get only the requests you received.
Mmmmh.

or maybe, in your model, you'll need something like:
```
has_many :received_requests, class_name: 'requests', through: :recievers

```